src/test/kotlin/com/terraformation/backend/db/ProjectInternalRoleTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.terraformation.backend.db
+
+import com.terraformation.backend.customer.shouldBeTfContact
+import com.terraformation.backend.db.default_schema.ProjectInternalRole
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+
+internal class ProjectInternalRoleTest {
+
+  @Test
+  fun `role should be tf contact`() {
+    assertTrue { ProjectInternalRole.ProjectLead.shouldBeTfContact() }
+    assertTrue { ProjectInternalRole.RestorationLead.shouldBeTfContact() }
+    assertFalse { ProjectInternalRole.Consultant.shouldBeTfContact() }
```

Comment:
These assertions (or at least the two `assertTrue` ones) should have messages so we know which one failed without having to look at the line numbers in the stack traces.